### PR TITLE
Filter the "content" attribute value of the <meta /> field

### LINKED ISSUE #99: Provide Filter to Meta Description and other meta fields.
I am trying to modify my meta description from my theme and unable to modify, since we are having
`$this->meta_field( 'description', $meta_description )` at line [553](https://github.com/alleyinteractive/wp-seo/blob/master/php/class-wp-seo.php#L553)

**My scenario**
if I have Title, use Title else uses meta description.

From the above code, At line no 541 I can only apply filter `wp_seo_meta_description_format` if I don't have a meta description.

But in my case, I need both since I am using meta description for og:description.

So, I decided to made changes in [File](https://github.com/alleyinteractive/wp-seo/blob/master/php/class-wp-seo.php) at `meta_field` method, so then we can modify any meta tag content easily using the filter.

**My code:** at line 500 will be 
`$content = apply_filters( 'wp_seo_meta_field', $content, $name );`

So, I can easily apply my filters to any meta tags used by WP-SEO like
`function meta_feild_content($content, $name){ /* mycode goes here */
return $content;
}`
`add_filter( 'wp_seo_meta_field', 'meta_field_content', 10, 2 ); `

Since I don't have permission it accesses this repo, I am unable to push the code.

If you think my approach is good can you ad the code.

Thanks
Arun

### PR DESCRIPTION
**Pull request by dlh01**
Fixes #99.

### CODE REVIEW

File: php/class-wp-seo.php [hunk 1]
```diff
@@ -493,14 +493,28 @@ public function pre_get_document_title( $title ) {
...
 
+			/**
+			 * Filters the "content" attribute value of the <meta /> field.
+			 *
+			 * @since 0.13.0
+			 *
+			 * @param string $content The "content" attribute value.
+			 * @param string $name    The "name" attribute value.
+			 */
+			$content = apply_filters( 'wp_seo_meta_field_content', $content, $name );
```
Comment (achaitanyajami) [inline php/class-wp-seo.php]: Can we move this filter up in the first line of the function?

So that we can have if condition next.

Comment (dlh01) [inline php/class-wp-seo.php]: I put it here to ensure the variables passed to the filter are strings. Does that cause a problem for your use case?

Comment (dlh01) [inline php/class-wp-seo.php]: (If so, then, yes, it definitely could be moved.)

Comment (achaitanyajami) [inline php/class-wp-seo.php]: Nope, not at all. But my thought is instead of using 2 if condition, we can go with single if condition.

Comment (dlh01) [inline php/class-wp-seo.php]: Yeah, let's go with the single check. Thanks!

**Pull request merged**